notebook/agentchats.ipynb

```diff
@@ -0,0 +1,993 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Auto Generated Agent Chat: Multi Task Chats\n",
```

Comment:
This title is confusing. How about:

"Solving a set of tasks with a sequence of chats"